test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java

```diff
@@ -70,8 +70,13 @@ public void initializeDeployment() {
 
     @After
     public void cleanEnvironment() {
-        deploymentScenario.undeploy();
-        gitProvider.deleteGitRepository(deploymentScenario.getNamespace());
+        if (deploymentScenario != null) {
+            deploymentScenario.undeploy();
+        }
+
+        if (gitProvider == null) {
```

Comment:
looks like != should be used here

Reply:
And also would have been identified by FindBugs.

Reply:
@triceo Just out of curiosity: what is your personal workflow regarding FindBugs? Do you scan your codebase from time to time and fix the warnings? Or before every commit? Of is it part of the build (fail on warning)? I'm thinking of a good way how to incorporate it into my workflow..

Reply:
@jhrcek As a general practice, I always run one final `mvn clean install` before I push a finished feature branch. Therefore, if Findbugs is integrated in the Maven build, it will have been run at that point.

Reply:
(I find it is useful to do that near the final push - if you have your code finalized, and your test coverage is complete, having Findbugs find new problems is actually a good sign that your coverage is not as complete as you thought. :-))